Example/Backpack/NavigationData.swift

```diff
@@ -20,6 +20,34 @@
 class NavigationData: NSObject {
     static var mainStoryboard = loadStoryboard(name: "Main")
 
+    static func enrich<R>(_ fun: @escaping () -> R, _ map: @escaping (R) -> Void) -> () -> R {
```

Comment:
I think function could move elsewhere, it's not related to this domain specifically. Maybe we can also add some docs to it?

Reply:
I moved it to `Presentable.swift`. Does that work?